pkg/awsutils/awsutils.go

```diff
@@ -439,13 +441,13 @@ func (cache *EC2InstanceMetadataCache) initWithEC2Metadata(ctx context.Context)
 	}
 	log.Debugf("%s is the primary ENI of this instance", cache.primaryENI)
 
-	// retrieve sub-id
-	cache.subnetID, err = cache.imds.GetSubnetID(ctx, mac)
```

Comment:
Do we no longer need to cache `subnetID`? If so, can we remove it from the data structure?

Reply:
Yes we no longer need it